static/usage/v7/modal/controller/demo.html

```diff
@@ -58,7 +57,7 @@
         const { data, role } = await modal.onWillDismiss();
 
         if (role === 'confirm') {
-          document.querySelector('#message').innerHTML = `Hello, ${data}!`;
```

Comment:
I wonder if maybe these modal ones felt more natural before the change? It's fine either way, but I kind of actually expected this one to be displayed in the DOM instead of the console. I feel like it's a common pattern for this sort of example.

Reply:
I was sort of wondering the same, to be honest 🤔 I only changed them for consistency's sake. I'll go ahead and revert them.